src/makeRobotAccountant.js

```diff
@@ -6,7 +6,26 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let sum = 0;
+  let isEven = 0;
```

Comment:
Your `sum` and `isEven` are equal. Why do you need both of them? One is redundand